packages/usa-form/src/test/test-pattern/test-usa-form.twig

```diff
@@ -204,8 +204,8 @@
     {% if error_state == true %}<span class="usa-error-message" role="alert">Helpful error message</span>{% endif %}
     <div class="usa-input-group {% if error_state == true %}usa-input--error{% endif %}">
       <div class="usa-input-prefix" aria-hidden="true">
-        <svg aria-hidden="true" role="img" focusable="false" class="usa-icon">
-          <use xlink:href="./img/sprite.svg#credit_card"></use>
+        <svg aria-hidden="true" role="img" focusable="false" class="usa-icon" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
        <svg aria-hidden="true" role="img" focusable="false" class="usa-icon">
```

Reply:
Resolved in [4dbefec](https://github.com/uswds/uswds/pull/6165/commits/4dbefecd5d272c477ae3908a5ee7c49f2657145f)

Reply:
It looks like 4dbefec removed the xlmns link from `packages/usa-modal/src/test/template.html` and not this file. Unresolving until that gets removed!

Reply:
Oops! Good catch(: Resolved in [4dbefec](https://github.com/uswds/uswds/pull/6165/commits/4dbefecd5d272c477ae3908a5ee7c49f2657145f)